resources/services/directconnect/gateways.go

```diff
@@ -16,10 +16,10 @@ func DirectconnectGateways() *schema.Table {
 		Name:          "aws_directconnect_gateways",
 		Description:   "Information about a Direct Connect gateway, which enables you to connect virtual interfaces and virtual private gateway or transit gateways.",
 		Resolver:      fetchDirectconnectGateways,
-		Multiplex:     client.ServiceAccountRegionMultiplexer("directconnect"),
+		Multiplex:     client.AccountMultiplex,
 		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
-		DeleteFilter:  client.DeleteAccountRegionFilter,
-		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "region", "id"}},
+		DeleteFilter:  client.DeleteAccountFilter,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
```

Comment:
Can we just use ARN instead?

this is the pattern for ARN:
`arn:${Partition}:directconnect::${Account}:dx-gateway/${DirectConnectGatewayId}`

Reply:
We have many resources where using the ARN instead would/might make sense, maybe a bigger discussion.